.github/workflows/test.yaml

```diff
@@ -0,0 +1,42 @@
+name: Tests
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+jobs:
+  tests:
+    runs-on: ubuntu-20.04
+    env:
+      GOPATH: ${{ github.workspace }}
+    steps:
+    - name: Set up Go 1.9
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.9
```

Comment:
perhaps it'd be useful to set up a strategy with go 1.9 and maybe the latest branch. Hopefully we'll move snapd to 1.13 soon, so it can replace 1.9 in this workflow too.